### PR TITLE
add snappy restriction note

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/15241
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/troubleshoot-tidb-lightning.md [hunk 1]
```diff
@@ -213,4 +213,8 @@ header = false
 
 ### `invalid compression type ...`
 
-TiDB v6.4.0 及之后版本的 TiDB Lightning 不支持带有非 `.bak` 后缀的数据文件并报错。你需要提前修改文件名，或将该类文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
+TiDB v6.4.0 及之后版本的 TiDB Lightning 仅支持带有 `.bak` 后缀的数据文件，以及 `gzip`、`snappy`、`zstd` 格式的压缩文件。其他类型的文件会报错。你需要提前修改文件名，或将该类文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
```
Comment (hfxsd) [inline tidb-lightning/troubleshoot-tidb-lightning.md:216]:
```suggestion
TiDB v6.4.0 及之后版本的 TiDB Lightning 仅支持 `gzip`、`snappy`、`zstd` 格式的压缩文件。其他类型的文件会报错。你需要提前修改文件名，或将该类文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
```

Comment (okJiang) [inline tidb-lightning/troubleshoot-tidb-lightning.md:216]:
```suggestion
TiDB v6.4.0 及之后版本的 TiDB Lightning 仅支持 `gzip`、`snappy`、`zstd` 格式的压缩文件。其他类型的压缩文件会报错。你可以修改文件名为 `.bak`，或将该类文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
```

File: tidb-lightning/tidb-lightning-data-source.md [hunk 2]
```diff
@@ -292,6 +292,7 @@ TiDB Lightning 目前支持由 Dumpling 导出的压缩文件或满足符合上
 > - TiDB Lightning 仅支持导入各自独立压缩的数据文件，不支持导入多个数据文件组成的单个压缩文件集合包。
 > - TiDB Lightning 不支持二次压缩的 `parquet` 文件，例如 `db.table.parquet.snappy`。如需压缩 `parquet` 文件，你可以配置 `parquet` 文件数据存储的压缩格式。
 > - TiDB v6.4.0 及之后版本的 TiDB Lightning 支持后缀为压缩算法 `gzip`、`snappy` 、`zstd`，以及后缀名 `.bak` 的数据文件。其他后缀名会报错。你需要提前修改文件名，或将该类文件移出导入数据目录来避免此类错误。
```
Comment (okJiang) [inline tidb-lightning/tidb-lightning-data-source.md:294]: ditto

**Review by okJiang — COMMENTED** [review]
除了 Lightning 和 Import into，其他文件貌似不需要修改

File: tidb-lightning/troubleshoot-tidb-lightning.md [hunk 3]
```diff
@@ -213,4 +213,8 @@ header = false
 
 ### `invalid compression type ...`
 
-TiDB v6.4.0 及之后版本的 TiDB Lightning 不支持带有非 `.bak` 后缀的数据文件并报错。你需要提前修改文件名，或将该类文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
+TiDB v6.4.0 及之后版本的 TiDB Lightning 仅支持 `gzip`、`snappy`、`zstd` 格式的压缩文件。其他类型的压缩文件会报错。你可以修改文件名为 `.bak`，或将该类文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
```
Comment (okJiang) [inline tidb-lightning/troubleshoot-tidb-lightning.md:216]:
```suggestion
TiDB v6.4.0 及之后版本的 TiDB Lightning 仅支持 `gzip`、`snappy`、`zstd` 格式的压缩文件。其他类型的压缩文件会报错。你可以修改文件名后缀为 `.bak`，或将该类文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
```

Comment (hfxsd) [inline tidb-lightning/troubleshoot-tidb-lightning.md:216]:
```suggestion
TiDB v6.4.0 及之后版本的 TiDB Lightning 仅支持 `gzip`、`snappy`、`zstd` 格式的压缩文件。其他类型的压缩文件会报错。你可以将该类文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
```

File: tidb-lightning/tidb-lightning-data-source.md (hunk 2 above)
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-data-source.md:294]:
```suggestion
> - TiDB v6.4.0 及之后版本的 TiDB Lightning 支持后缀为压缩算法 `gzip`、`snappy` 、`zstd`。其他后缀名会报错。你可以将不支持的文件移出导入数据目录来避免此类错误。
```

File: tidb-lightning/troubleshoot-tidb-lightning.md [hunk 4]
```diff
@@ -213,4 +213,8 @@ header = false
 
 ### `invalid compression type ...`
 
-TiDB v6.4.0 及之后版本的 TiDB Lightning 不支持带有非 `.bak` 后缀的数据文件并报错。你需要提前修改文件名，或将该类文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
+TiDB v6.4.0 及之后版本的 TiDB Lightning 仅支持 `gzip`、`snappy`、`zstd` 格式的压缩文件。其他类型的压缩文件会报错。你可以将不支持的压缩文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
```
Comment (Frank945946) [inline tidb-lightning/troubleshoot-tidb-lightning.md:216]:
```suggestion
TiDB v6.4.0 及之后版本的 TiDB Lightning 仅支持 `gzip`、`snappy`、`zstd` 格式的压缩文件。如果存放源数据文件的目录下存在未支持的压缩文件会导致任务报错。你可以将不支持的压缩文件移出导入数据目录来避免此类错误。更多详情请参考[压缩导出的数据文件](/tidb-lightning/tidb-lightning-data-source.md#压缩文件)。
```

**Review by Frank945946 — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]
rest lgtm

**Pull request merged**